databricks/koala/tests/test_dataframe.py

```diff
@@ -249,11 +277,7 @@ def test_value_counts(self):
         df = pd.DataFrame({'x': [1, 2, 1, 3, 3, np.nan, 1, 4]})
         ddf = self.spark.from_pandas(df)
 
-        exp = df.x.value_counts()
```

Comment:
I reverted a PR that was causing problems and I think you forked the code at that moment. Can you merge with the most recent master? You should not have to include these changes.

Reply:
Fixed.